Use weaveworks/flux-get-started repo in docs (#1527)
weaveworks/flux: chart/flux/README.md

# Git repo

>  - One repo containing desired release state information and Charts themselves.

Reviewer: ```suggestion
 - One repo containing cluster config (i.e., Kubernetes YAMLs) and zero or more git repos containing Charts themselves.
```

>  - Charts are co-located under another path or can be referenced from Helm repositories.

Reviewer: ```suggestion
 - Charts can be co-located with config in the git repo, or be from Helm repositories.
```

| `helmOperator.chartsSyncInterval` | Interval at which to check for changed charts | `3m`
| `helmOperator.chartsSyncTimeout` | Timeout when checking for changed charts | `1m`

> | `helmOperator.extraEnvs` | Extra environment variables for the Helm operator pod | `[]`

Author: my bad... I put it back
